namespaces/cloud-platform-test-1.k8s.integration.dsd.io/platforms-dev/resources/main.tf

```diff
@@ -0,0 +1,17 @@
+provider "aws" {
```

Comment:
Should we add the backend config here?
This will be the first example of how to use terraform-managed resources so I think we should manage the state as well.